src/bootstrap-table.js

```diff
@@ -1602,9 +1605,6 @@ class BootstrapTable {
 
         item[this.header.stateField] = value === true || (!!value_ || (value && value.checked))
       } else {
-        value = typeof value === 'undefined' || value === null ?
```

Comment:
why do you removed that ?

Reply:
removed as it is now on line 1562 as we need value to be !== null for line 1566